Content/NPCs/Misc/MagnetizedEnemies.cs

```diff
@@ -0,0 +1,278 @@
+﻿using StarlightRiver.Content.Items.Dungeon;
+using StarlightRiver.Content.Items.Magnet;
+using StarlightRiver.Core;
+using StarlightRiver.Helpers;
+using System.Collections.Generic;
+using System.Linq;
+using Terraria;
+using Terraria.DataStructures;
+using Terraria.ID;
+using Terraria.Graphics.Effects;
+using System;
+using StarlightRiver.Content.Dusts;
+
+namespace StarlightRiver.Content.NPCs.Misc
+{
+    public class MagnetizedEnemies : GlobalNPC
```

Comment:
Should this maybe be named MagnetizedEnemy (singular)? This modifies the behavior of individual enemies, not a seperately maintained collection of them

Reply:
Fixed